includes/admin.php

```diff
@@ -184,21 +174,18 @@ function get_help_text( $screen = null, $return_array = true ) {
 			'document' => array(
 				__( 'Basic Usage', 'wp-document-revisions' ) =>
 				'<p>' . __( 'This screen allows users to collaboratively edit documents and track their revision history. To begin, enter a title for the document, click <code>Upload New Version</code> and select the file from your computer.', 'wp-document-revisions' ) . '</p>' .
-				'<p>' . __( 'Once successfully uploaded, you can enter a revision log message, assign the document an author, and describe its current workflow state.', 'wp-document-revisions' ) . '</p>' .
+				'<p>' . __( 'Once successfully uploaded, you can enter a revision log message, assign the document an author.', 'wp-document-revisions' ) . '</p>' .
```

Comment:
"Once successfully uploaded, you can enter a revision log message **and** assign the document an author."